src/session.ts

```diff
@@ -551,8 +550,6 @@ async function warmTransfer(
 /*--------------------------------------------------------------------------------------------------------------------*/
 
 async function transfer(this: WebPhoneSession, target: WebPhoneSession, options): Promise<ReferClientContext> {
-    await (this.localHold ? Promise.resolve(null) : this.hold());
-    await delay(300);
```

Comment:
Same here, is it safe to not put the call on hold before blind transfer?

Reply:
The issue is that hold might not be finished in 300ms, this logic trigger a bug in Jupiter when app send REFER before the reInvite is success, server will return success for the REFER but do nothing. 

In mThor, we won't hold call before we transfer / warmTransfer it.  I also capture the SIP for Spartan and spartan will also NOT hold call before transfer call